### ISSUE TITLE
An `octave-workspace` file is created even if the session is properly terminated using `exit()`

### ISSUE THREAD

**Issue by truenicoco** [blink1073/oct2py#133]
The title pretty much sums it up. Although this is most definitely a minor issue for me, it is annoying because it creates a lot of files I don't need or want.

Octave version 4.2.1
oct2py version 4.0.6

Illustration in iPython:
```
In [23]: octave = Oct2Py()

In [24]: testarray = octave.eval("zeros(50);")

In [25]: octave.exit()

In [26]: ls oct*
octave-workspace
```

**Comment by ppebay** [comment]
@truenicoco @blink1073 

Hello -- any update regarding this issue? I am observing the same problem with oct2py (obtained from MacPorts).

Thank you

**Comment by truenicoco** [comment]
Unfortunately no, I'm still deleting a lot of these files manually.

**Comment by ppebay** [comment]
@truenicoco @blink1073

Thanks for the update. Hopefully the package manager is looking at the issues here.

**Comment by blink1073** [comment]
Fixed in 4.2.0.
